src/main/java/jdwp/JDWPStackFrame.java

```diff
@@ -115,6 +118,32 @@ public void reply(GDBControl gc, PacketStream answer, PacketStream command) {
                                     answer.writeNullObjectRef();
                             }
                         }
+                    } else if (vmVar.name().equals("$asm")) { // To do: create a new StringReferenceImpl
+                        ObjectReferenceImpl strObject = JDWP.strRef.instances(10000).get(0);
+                        StringReferenceImpl newStrRef = new StringReferenceImpl(strObject.referenceType(), null, "myStr");
+                        JDWP.strRef.instances(1).add(newStrRef);
+                        answer.writeByte(tag);
+//                        answer.writeObjectRef(newStrRef.uniqueID());
```

Comment:
if we aren't using the string ref stuff yet, can we remove all that from the PR? You should leave them in your branch for experiments till we get it to work.